File: src/WebApp/src/app/sales/interfaces/response-invoice-sale.ts
import {InvoiceSale} from './invoice-sale';
import {InvoiceSaleDetail} from './invoice-sale-detail';
import {TributoSale} from './tributo-sale';

// respuesta comprobante de venta.
export class ResponseInvoiceSale {
  constructor(
    public invoiceSale: InvoiceSale = new InvoiceSale(),
    public invoiceSaleDetails: Array<InvoiceSaleDetail> = new Array<InvoiceSaleDetail>(),
    public tributoSales: Array<TributoSale> = new Array<TributoSale>()) {
  }
}

File: Data/ViewModels/Sales/ResponseInvoiceSale.cs
using Nebula.Data.Models.Sales;

namespace Nebula.Data.ViewModels.Sales;

public class ResponseInvoiceSale
{
    public InvoiceSale? InvoiceSale { get; set; }
    public List<InvoiceSaleDetail>? InvoiceSaleDetails { get; set; }
    public List<TributoSale>? TributoSales { get; set; }
}
